src/styles/blocks/nav.scss

```
@@ -0,0 +1,50 @@
+.nav {
+  display: flex;
```

display flex is abundant on nav, you've already added it to list
```suggestion
.nav {
```